wicamo96/Verti-Comps
# Authentication

## What Changed

- Added components allowing you to register a new user or login as an existing user
- Added Views components to allow routing of the application to register or login

## How To Test

- You should be directed to login page
- Login using the preexisting login credentials
- Check that the user's id and staff boolean is stored in the application tab of dev tools and that they are correct
- Delete that info from the application tab, refresh the page, and enter different login credentials from the db
- Check and delete that info same as above
-----------------------------------------------------------------------
- Test the register page by inputting your own info
- You should see all info posted to db if if was correctly entered
- Test with and without administrator button
